sale_import_base/tests/common_sale_order_import.py

```diff
@@ -0,0 +1,202 @@
+            "street2": "bis",
+            "zip": "69100",
+            "city": "Lyon",
+            "email": "whatever@whatever.io",
+            "country_code": "FR",
+        }
+
+    @classmethod
+    def setUpLines(cls):
+        cls.line_valid_1 = {
+            "product_code": "PROD_ORDER",
+            "qty": 5,
+            "price_unit": 1111.1,
+            "description": "Initial Line 1 import description",
+            "discount": 10.0,
+        }
+        cls.line_valid_2 = {
+            "product_code": "PROD_DEL",
+            "qty": 2,
+            "price_unit": 2222.2,
+            # description: missing
+            "discount": 0.0,
+        }
+        cls.line_invalid = {
+            "product_code": "DOES NOT EXIST",
+            "qty": "should not happen",
+            "price_unit": "wrong input",
+            "description": "",
+            "discount": 0.0,
+        }
+
+    @classmethod
+    def setUpImport(cls):
+        amt_untaxed_1 = (
+            cls.line_valid_1["price_unit"]
+            * cls.line_valid_1["qty"]
+            * (1 - cls.line_valid_1["discount"] / 100)
+        )
+        amt_tax_1 = amt_untaxed_1 * 0.09
+        amt_total_1 = amt_untaxed_1 + amt_tax_1
+        amt_untaxed_2 = cls.line_valid_2["price_unit"] * cls.line_valid_2["qty"]
+        amt_tax_2 = amt_untaxed_2 * 0.00
+        amt_total_2 = amt_untaxed_2 + amt_tax_2
+        cls.amount_valid = {
+            "amount_tax": amt_tax_1 + amt_tax_2,
+            "amount_untaxed": amt_untaxed_1 + amt_untaxed_2,
+            "amount_total": amt_total_1 + amt_total_2,
+        }
+        cls.amount_invalid = {
+            "amount_tax": "this should not",
+            "amount_untaxed": "happen",
+            "amount_total": dict(),
+        }
+        cls.sale_order_invoice_example = {"date": "1900-12-30", "number": "IN-123"}
+        cls.sale_channel_ebay = cls.env.ref("sale_channel.sale_channel_ebay")
+        cls.sale_order_example_vals = {
+            "address_customer": cls.addr_customer_example,
+            "address_shipping": cls.addr_shipping_example,
+            "address_invoicing": cls.addr_invoicing_example,
+            "lines": [cls.line_valid_1, cls.line_valid_2],
+            "amount": cls.amount_valid,
+            "transaction_id": 123,
+            "invoice": cls.sale_order_invoice_example,
+            "sale_channel": cls.sale_channel_ebay.name,
+            "currency_code": "USD",
+            "pricelist_id": cls.env.ref("product.list0").id,
+        }
+
+    @classmethod
+    def setUpTaxes(cls):
+        Tax = cls.env["account.tax"]
+        tax_vals = {
+            "name": "Fictional tax 9%",
+            "amount": "9.00",
+            "type_tax_use": "sale",
+            "company_id": cls.env.ref("base.main_company").id,
+        }
+        cls.tax = Tax.create(tax_vals)
+        cls.product_order.taxes_id = cls.tax
+
+    @classmethod
+    def setUpFpos(cls):
+        Tax = cls.env["account.tax"]
```

Comment:
please follow oca convention for variable naming https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#variable-names